src/api/app/controllers/webui/image_templates/interconnects_controller.rb

```diff
@@ -0,0 +1,12 @@
+module Webui
+  module ImageTemplates
+    class InterconnectsController < Webui::WebuiController
+      def index
+        @projects = Project.image_templates
+        respond_to do |format|
+          format.xml
```

Comment:
Anyway, what's the reasoning for having a seperate webui controller for the xml representation of image templates? It just happen to be a used by an "interconnect"....